Simplenote/Classes/SPSettingsViewController.swift

```diff
@@ -0,0 +1,43 @@
+import UIKit
```

Comment:
Nipticky: Mind renaming this file as `+Extensions` so that it's 100% no doubt it does not implement the full ViewController?